doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -1196,6 +1196,130 @@ Abbreviations
    denoted expression is performed at definition time. Type-checking is
    done only at the time of use of the abbreviation.
 
+
+Numeral notations
+-----------------
+
+Starting with version 8.9, the following command allows the user to customize
+the way numeral literals are parsed and printed:
+
+.. cmd:: Numeral Notation @ty @pa @pr : @scope.
```

Comment:
`@stuff` is supposed to denote a grammar entry. This is not the case here for `@ty` `@pa` `@pr`. Use `@ident__1` `@ident__2` and `@ident__3` instead.